dlrs/main/classes/RollupSummariesSelector.cls

```diff
@@ -527,29 +375,29 @@ public class RollupSummariesSelector {
     ) {
       return LookupRollupSummary2__mdt.getInstance(developerNameOrId);
     }
+  }
 
-    private List<RollupSummary> sortSummaries(
-      List<RollupSummary> summaries,
-      String sortField,
-      String sortFieldSecondary
-    ) {
-      List<RollupSummaryComparable> rollupSummaryComparables = new List<RollupSummaryComparable>();
-      for (RollupSummary summary : summaries) {
-        rollupSummaryComparables.add(
-          new RollupSummaryComparable(summary, sortField, sortFieldSecondary)
-        );
-      }
-
-      rollupSummaryComparables.sort();
+  private static List<RollupSummary> sortSummaries(
```

Comment:
sortSummaries moved to the outer class scope so it can be access but the new logic to pull from in memory data.